jest-puppeteer.config.js

```diff
@@ -0,0 +1,18 @@
+// https://github.com/smooth-code/jest-puppeteer/blob/master/packages/jest-environment-puppeteer/README.md#jest-puppeteerconfigjs
```

Comment:
Let's not override this completely. Instead, lets extend the default config and only add the parts we need:
https://github.com/WordPress/gutenberg/blob/7060b11b9b9efc00e099c9092fe520402d77ee4b/packages/scripts/config/puppeteer.config.js

We can import it from `@wordpress/scripts/config/puppeteer.config.js`

Reply:
Yep, updated.